### PR TITLE
Add glob support for self hosted runner known labels configuration

### PR DESCRIPTION
**Pull request by kishaningithub**
Addresses #359 and #377

### CODE REVIEW

**Comment by dekimsey** [comment]
Thank you for this @kishaningithub, this will address our ability to lint with custom runners! 👍

File: .gitignore
Comment (jenseng) [inline .gitignore:16]: I'll let the maintainers chime in, but typically I'd put this sort of thing in my own global gitignore; people develop in all kinds of environments, so adding that kind of stuff to a project-level gitignore can make it unwieldy

**Review by rhysd — APPROVED** [review]
Nice addition. Thank you.

**Pull request merged**
